src/test/java/com/baeldung/selenium/alertsandpopups/HandleAlertsUnitTests.java

```diff
@@ -0,0 +1,140 @@
+package com.baeldung.selenium.alertsandpopups;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.time.Duration;
+
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoAlertPresentException;
+import org.openqa.selenium.UnhandledAlertException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class HandleAlertsUnitTests {
+
+    @Test
+    public void whenSimpleAlertIsTriggered_thenHandleIt() {
+        WebDriver driver = new ChromeDriver();
+        driver.get("https://testpages.herokuapp.com/styled/alerts/alert-test.html");
+        driver.findElement(By.id("alertexamples"))
+            .click();
+
+        Alert alert = driver.switchTo()
+            .alert();
+        String alertText = alert.getText();
+        alert.accept();
+        assertEquals("I am an alert box!", alertText);
+        driver.quit();
```

Comment:
This should be common to all tests (in an after block) to ensure the driver is quit even if the test fails